src/hooks/useScanApi.ts

```diff
@@ -259,10 +259,9 @@ const useDeleteScanApi = (onAddAlert: (alert: AlertProps) => void) => {
 
 const useGetScanJobsApi = (onAddAlert: (alert: AlertProps) => void) => {
   const { t } = useTranslation();
-
```

Comment:
I think that if you keep that line, then you won't need any changes in `tests/__snapshots__/code.test.ts.snap`. That other file is common source of merge conflicts.

Does linter complain about this empty line?